src/lib.rs

```diff
@@ -15,6 +15,7 @@
 //! - `parity` — [`parity-scale-codec`][parity_scale_codec] support (`Encode` and `Decode`
 //!   implementations).
 //! - `serde` — support for `serde`. Enabled by default.
+//! - `schemars` — support for `schemars`. Enabled by default.
```

Comment:
It's disabled by default. Hmm, `serde` is also disabled by default.

Reply:
Edited comment